src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala

```diff
@@ -194,6 +194,9 @@ class JsonRpcController(
 
     case req @ JsonRpcRequest(_, "personal_deleteWallet", _, _) =>
       handle[DeleteWalletRequest, DeleteWalletResponse](personalService.deleteWallet, req)
+
+    case req @ JsonRpcRequest(_, "personal_changePassphrase", _, _) =>
```

Comment:
I think we need to discuss if new endpoints will be under existing _namespaces_ or not. This is the second case (see https://github.com/input-output-hk/mantis/pull/330)

I don't have a strong opinion tho.

Reply:
> I don't have a strong opinion tho.

Me neither. I was hoping the review would clarify that 😄

Reply:
Changed to `daedalus`